src/website/index.css

```diff
@@ -1,7 +0,0 @@
-body {
-    margin: 0;
-}
-pre.prism-code {
```

Comment:
I think these styles are only no longer needed when there is no sidebar, otherwise it squeezes the sidebar to be too narrow.

Rather than making them global, though, maybe we should make a `Pre` component in the site, and use that in `ComponentDocExample` and `Markdown` (those are the only places using codeblocks, I think).

Reply:
Good suggestion to create a new component to take it a step further.  I may try it if it does not blow up the scope.

Reply:
I still think that the word-wrap/line-break styles should be removed.  It really helps legibility.  

The issue with the sidebar likely needs resolved differently as the current styles also cause problems with the Popover/boundary demo which uses the ScrollBox component.  It currently appears to be due the main layout's flexbox configuration.  Perhaps there is a fix where we can continue to use flexbox; alternatively we may need to fall back to floats.

I will happily demo this for you to make things more clear.